csharp/NedapRetailApiClient/Nedap.Retail.Api/Epc.V2/DifferenceListResponse.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Nedap.Retail.Api.Epc.V2
+{
+    /// <summary>
+    /// Difference between ERP system stock and RFID stock
+    /// </summary>
+    public class DifferenceListResponse : JsonPrintableObject
+    {
+        /// <summary>
+        /// The GTINs in the difference list.
+        /// </summary>
+        [JsonProperty("gtins")]
+        public List<string> Gtins { get; set; }
```

Comment:
why allow set on a response object? A response object should be immutable in my opinion.